Affiliate program.md

```diff
@@ -0,0 +1,25 @@
+Below is what I managed to prepare for a few hours with the ChatGPT and my previous research.
+
+The topic needs further discussion.
+
+**Mechanism for creating value:**
+- Companies (AWS, Notion, etc) pay affiliate commission to partners for attracting clients. The commission is from 5% to 70% of the purchase amount.
+- Ubiquity can become a partner of such companies and transfer affiliate commission to bounty hunters.
+- For this bounty, hunters must pay with tokens.
+ The token can be UBQ or a new one - UXP (Ubiquity Experience), KARMA (karma of the developer), etc.
+
+**Mechanism for obtaining tokens:**
+- As a bonus for the efficient completion of tasks by bounty hunters.
```

Comment:
I don't think that experience should be transferable though, so this should just be saved in our database and associated with the GitHub user.